libvirt/tests/src/snapshot/create_manual_snapshot.py

```
@@ -0,0 +1,254 @@
+    :param test: test object
+    """
+    test.log.info("TEST_STEP3: Create snapshots based on the snapshot command.")
+    vm_name = params.get("main_vm")
+    snap_name = params.get("snap_name")
+    snap_options = params.get("snap_options")
+    diskspec_options = params.get("diskspec_options")
+
+    options_parts = [snap_name, snap_options, diskspec_options]
+    full_snap_options = " ".join(part for part in options_parts if part)
+    virsh.snapshot_create_as(vm_name, full_snap_options, **virsh_dargs)
+    snapshot_list = virsh.snapshot_list(vm_name, **virsh_dargs)
```

Ruff (0.14.11)

Local variable `snapshot_list` is assigned to but never used

Remove assignment to unused variable `snapshot_list`

(F841)